src/components/v2/Table/TableContainer/TableContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { forwardRef } from 'react'
+import { Dropdown } from '../../../../'
+import { Tabs } from '../../../Tabs/Tabs'
+import { Props } from './TableContianer.types'
+import './TableContainer.css'
+
+const TableContainer = forwardRef<HTMLDivElement, Props>((props, ref) => {
+  const {
+    children,
+    tabsList,
+    activeTab,
+    handleTabChange,
+    sortbyList,
+    handleSortByChange,
+    sortBy
+  } = props
+
+  return (
+    <div className="dui-table-container" ref={ref}>
+      {tabsList.length || sortbyList?.length ? (
+        <div className="filtertabsContainer">
+          {tabsList.length > 0 ? (
+            <Tabs isFullscreen>
+              {tabsList.map((tab) => (
+                <Tabs.Tab
+                  key={tab.value}
+                  active={activeTab === tab.value}
+                  onClick={() => {
+                    handleTabChange && handleTabChange(tab.value)
+                  }}
+                >
+                  <div className="tabStyle">{tab.displayValue}</div>
+                </Tabs.Tab>
+              ))}
+            </Tabs>
+          ) : null}
+          {sortbyList && (
+            <Dropdown
+              direction="left"
+              className="sortByDropdown"
+              value={sortBy}
+              onChange={(_event, data) => {
+                const value = data.value as string
```

Comment:
To be sure that the data is a string, what do you think about using `toString` here?
```suggestion
                const value = data.value.toString()
```